From a126aee948b06e391cb0aa9f03eaa9bf80f276d2 Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Tue, 10 Jul 2018 15:29:22 +0800
Subject: [PATCH 0898/1443] drm/i915/gvt: enable pv ppgtt update by default

This patch enables pv ppgtt update by default.

It is needed for both uos and sos.

Signed-off-by: Zhipeng Gong <zhipeng.gong@intel.com>
Reviewed-by: He, Min <min.he@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/i915_params.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.h b/drivers/gpu/drm/i915/i915_params.h
index d9b901f..2e56ea5 100644
--- a/drivers/gpu/drm/i915/i915_params.h
+++ b/drivers/gpu/drm/i915/i915_params.h
@@ -70,7 +70,8 @@ struct drm_printer;
 	param(bool, enable_dpcd_backlight, false) \
 	param(int, domain_scaler_owner, 0x11100) \
 	param(unsigned int, enable_pvmmio, \
-			PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE) \
+			PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE \
+			| PVMMIO_PPGTT_UPDATE) \
 	param(bool, enable_gvt, false)
 
 #define MEMBER(T, member, ...) T member;
-- 
2.7.4

